distributedlog-core/src/main/java/org/apache/distributedlog/tools/DistributedLogTool.java

```diff
@@ -2854,7 +2856,7 @@ public DistributedLogTool() {
         addCommand(new ListCommand());
         addCommand(new ReadLastConfirmedCommand());
         addCommand(new ReadEntriesCommand());
-        addCommand(new RecoverCommand());
+        // addCommand(new RecoverCommand());
```

Comment:
I think this command depends on some auto reocvery changes that need to be ported back from twitter branch.

@zhaijack can you create a github issue and comment the issue in the code? we can fix it later.